ado/ipacheckreplace.ado

```diff
@@ -0,0 +1,47 @@
+
```

Comment:
@kelseylarson - Did you  commit this file by mistake? I know Caton said you were working on modifying `readreplace` to deal with `select_multiple` variables. If it's ready we should still move it to a separate commit/PR to improve version control.